public/locales/en-US/guide-flexfarmer.json

```diff
@@ -11,7 +11,9 @@
       { "text": "At least 128MB available RAM." },
       { "text": "Initialized Plot NFT (Made via GUI or CLI)." },
       { "text": "Plots bound to that Plot NFT." },
-      { "text": "PlotNFT joined to Flexpool (see getting started for instructions)." }
+      {
+        "text": "PlotNFT joined to Flexpool (To manage your PlotNFT with the Chia client, see our <a href='https://www.flexpool.io/get-started/xch/XCH-CLI/'>CLI</a> and <a href='https://www.flexpool.io/get-started/xch/XCH-GUI/'>GUI</a> instructions)."
```

Comment:
Can you please move `hrefs` to the Trans components section? And use `LinkOut` instead of `a` for links?

So you will just have `<guilink>` and `<clilink>` components by doing this:

```tsx
<li key={item.text}>
          <Trans
            components={{
              a: <a href="" target="__blank" />,
              guilink: <LinkOut href="https://www.flexpool.io/get-started/xch/XCH-GUI/" target="__blank" />,
              clilink: <LinkOut href="https://www.flexpool.io/get-started/xch/XCH-CLI/" target="__blank" />
            }}
          >
            {item.text}
          </Trans>
        </li>
```